1-mandatory/3-slideshow/slideshow.js

```diff
@@ -1 +1,68 @@
 // Write your code here
+let photo = document.querySelectorAll("img");
+let counter = 0;
+let moveInterval;
+let moveBackInterval;
+
+let btnAutoBack = document.getElementById("aback");
+let btnBack = document.getElementById("back");
+let btnStop= document.getElementById("stop");
+let btnForward = document.getElementById("forward");
+let btnAutForward = document.getElementById("aforward");
+
+photo.forEach((img, i) => {
+    img.style.left = `${i*100}%`;
+});
```

Comment:
Can you explain what is happening here?  Where are all the images being displayed?